_posts/2016-09-29-Rust-1.12.md

```diff
@@ -0,0 +1,341 @@
+---
+layout: post
+title: "Announcing Rust 1.12.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.12.0. Rust is
+a systems programming language with the slogan "fast, reliable, productive:
+pick three."
+
+As always, you can [install Rust 1.12.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.12.0][notes] on GitHub.
+1361 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1120-2016-09-29
+
+### What's in 1.12.0 stable
+
+The release of 1.12 might be one of the most significant Rust releases since
+1.0. We have a lot to cover, but if you don't have time for that, here's a
+summary:
+
+The largest user-facing change in 1.12 stable is the new error message format
+emitted by `rustc`. We've [previously talked] about this format and this is the
+first stable release where they are broadly available. These error messages are
+a result of the effort of many hours of [volunteer effort] to design, test, and
+update every one of `rustc`s errors to the new format. We're excited to see
+what you think of them:
+
+![A new borrow-check error](/images/2016-09-Rust-1.12/borrowck-error.png)
```

Comment:
In the preview this image looks hella big :)  We may want to scale it down a bit